app.js

```diff
@@ -233,6 +233,46 @@ $("menu_top_right_contents").addEventListener("click", event => {
 });
 
 //
+// UI events to set the font size
+//
+$("input_fontsize").value = Config.BookViewer.defaultFontSize;
+
+$("decrease_font_size").addEventListener("click", event => {
+  var font_size_input = $("input_fontsize");
```

Comment:
Looks like you should factor the code for `increase_font_size` and `decrease_font_size`.